sympy/physics/mechanics/beam.py

```diff
@@ -0,0 +1,263 @@
+"""
+This module can be used to solve beam bending problems in mechanics.
+
+"""
+
+from __future__ import print_function, division
+
+from sympy.printing import sstr
+from sympy.physics.mechanics import Point
+from sympy import sympify
+
+
+class Beam(object):
+
+    """
+    A Beam is a structural element that is capable of withstanding load
+    primarily by resisting against bending.
+    Beams are characterized by their profile (moment of Inertia), their length,
+    and their material.
+
+    Parameters
+    ==========
+    length : Sympifyable
+        A SymPy expression representing the Beam's length.
+    E : Sympifyable
+        A SymPy expression representing the Beam's Young's Modulus.
+        It is a measure of the stiffness of the Beam material.
+    I : Sympifyable
+        A SymPy expression representing the Beam's Moment of Inertia.
```

Comment:
E and I should be words not single letters. Maybe: "length", "elastic_modulus", and "second_moment".

E: "Young's Modulus", "Modulus of Elasticity", or "Elastic Modulus". The later are preferred.